.github/ISSUE_TEMPLATE.md

```diff
@@ -1,5 +1,9 @@
 If your issue is not following the template, the maintainer may close the issue without notifications.
 
+Issue Type: Ask Questions.
+---------------------------
+Write your question on the discussion forum instead create a new issue.
+https://github.com/orgs/embulk/discussions
```

Comment:
```suggestion

Issue Type: Questions
----------------------

Post your question in User forum at GitHub Discussions, instead of submitting an Issue here.

https://github.com/orgs/embulk/discussions

```

Reply:
Thanks! Fixed.